src/protocol.cpp

```diff
@@ -140,3 +172,9 @@ std::string CInv::ToString() const
 {
     return strprintf("%s %s", GetCommand(), hash.ToString());
 }
+
```

Comment:
Is this double newline supposed to be here? I noticed it earlier in the file too (unrelated to this change).